pkgs/development/compilers/odin/default.nix

```diff
@@ -38,6 +45,7 @@ in stdenv.mkDerivation rec {
     mkdir -p $out/bin
     cp odin $out/bin/odin
     cp -r core $out/bin/core
+    cp -r vendor $out/bin/vendor
```

Comment:
core and vendor are very generic binary names

Reply:
Those are not binaries, those are library folders that are imported by odin programs.

Unfortunately odin is in pretty early stages, and currently requires those folders to be next to the executable (ie. `bin`). I don't think we can do much about it without patching the source code.

Reply:
> As they are folders I do not believe they can cause name collisions, correct?

If you have a binary named vendor or core it should complain.

Can we move the binary somewhere else like share? Then we can have the directories next to it. Also bin should not contain directories.

Reply:
Here are the relevant lines in the binary source: https://github.com/odin-lang/Odin/blob/3a761395bee4f14c81682d4c21a5aa1e9565959a/src/main.cpp#L2598

Seems like it may be possible to change where odin searches for those libraries by wrapping the binary and setting the `ODIN_ROOT` environment (see https://github.com/odin-lang/Odin/blob/3a761395bee4f14c81682d4c21a5aa1e9565959a/src/build_settings.cpp#L697), I'm unsure though if it will have any other impact on functionality.

Reply:
Oh neat! I think wrapping the binary should work then.